docs/modules/ROOT/pages/how-to/day2ops/remove-organization.adoc

```diff
@@ -0,0 +1,26 @@
+= Remove an organization
+
+== Prerequisites
+
+* All namespaces belonging to that organisation, on every APPUiO Cloud zone, are removed.
+* https://kb.vshn.ch/appuio-cloud/how-to/day2ops/connect-control-api.html[Connect to control api]
+
+
+== Check for organization namespaces
+
+On every APPUiO Cloud zone:
+
+[source,bash]
+----
+kubectl get ns -ocustom-columns="NAME:.metadata.name,ORG:.metadata.labels.appuio\.io\/organization" | grep <ORG_NAME>
+----
```

Comment:
```suggestion
----

If any namespace is still present, contact the customer and make them delete it.
Alternatively, we can delete them but we need the agreement by the customer that it is OK.
Make them understand, that this will permanently delete all persistent volumes without a means to recover them.
```

I wonder what happens with K8up backups.